server/src/tenant/schema-builder/graphql-types/input/big-float-filter.type.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  GraphQLInputObjectType,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLFloat,
+} from 'graphql';
+
+import { FilterIsEnumType } from './filter-is-enum-filter.type';
+
+export const BigFloatFilterType = new GraphQLInputObjectType({
+  name: 'BigFloatType',
+  fields: () => ({
```

Comment:
It seems fields is not instantiate the same way comparing to other filters, is it intentional?

Reply:
No you're right, it doesn't need to be a function, I'll make if more clean